backend/src/models/reports/index.js

```diff
@@ -545,6 +545,38 @@ const getDashboardByPortfolios = (portfolios) => {
   return query;
 };
 
+/**
+ * Gets the data for a Divisional Active Projects Report
+ *
+ * @param   {number[]} portfolios Optional list of portfolio_ids to limit report to. If empty, returns data for all portfolios.
+ * @returns {any[]}
+ */
+const getActiveProjects = (portfolios) => {
```

Comment:
i worry that the naming convention of this doesn't reflect the thing, as it is portfolio_id.
the table name and variable although shouldn't cause any code to fail, i feel like it could confuse the reader.  Was their a reason why you didn't use portfolio_id, or portfolioId what ever convention we have been using.

Reply:
Hmm. Because it is actually a list of 0 or more portfolios. However I could see an argument that it should have been something like portfolioIDs.